pkg/virt-controller/services/template.go

```diff
@@ -858,6 +858,12 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 		v1.DomainAnnotation: domain,
 	}
 
+	if vmi.ObjectMeta.Annotations != nil {
+		for key, value := range vmi.ObjectMeta.Annotations {
```

Comment:
I have to join to @booxter here: are we sure that *all* vmi annotations are meaningful and harmless when applied onto the launcher Pod? I'm not convinced yet, and appreciate a separate PR with a proper explanation.

Reply:
no problem I will remove it 

@booxter I just use it to tell istio to use the service mesh only for cluster and pod ip's so I am able to install things like nginx from the yum repo

``` yam,
apiVersion: kubevirt.io/v1alpha3
kind: VirtualMachineInstance
metadata:
  labels:
    special: vmi-masquerade
  annotations:
    traffic.sidecar.istio.io/includeOutboundIPRanges: "10.96.0.0/12,10.244.0.0/24"
  name: vmi-masquerade
...
```

Reply:
@SchSeba if there is a use case where we may want to allow users to specify annotations for their pods in VMI specs (and we should analyze whether it's safe to do it; open ended features are often vectors for attack), then perhaps, instead of overloading the meaning of VMI-specific annotations, we could introduce a separate attribute specifically to carry custom annotations for pods. But let's discuss this in a separate venue.